public/app/features/alerting/unified/components/mute-timings/MuteTimingForm.tsx

```diff
@@ -100,7 +100,7 @@ const MuteTimingForm = ({ muteTiming, showError, loading, provisioned, editMode
       {provisioned && <ProvisioningAlert resource={ProvisionedResource.MuteTiming} />}
       <FormProvider {...formApi}>
         <form onSubmit={formApi.handleSubmit(onSubmit)} data-testid="mute-timing-form">
-          <FieldSet label={'Create mute timing'} disabled={provisioned || updating}>
```

Comment:
Removed this because
1. it shows two title on a page which is weird
2. it would show "create mute timing" also when editing one :)